.github/pull_request_template.md

```
@@ -0,0 +1,25 @@
+#### Description
+
+<!-- Add description for PR -->
+
+#### Deployment URL: <URL>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: URL

(MD033, no-inline-html)